server/cli/prepare.py

```diff
@@ -172,6 +182,7 @@ def run_layout(adata):
 
     def show_step(item):
         names = {
+            "calculate_qc_metrics": "Calculating QC metrics" if run_qc else "Skipping QC",
```

Comment:
I'm not a fan of inline if else's in python code. The exceptions being in list comprehension and lambda functions. Could you move this to an if statement above names?